src/languages/en.ts

```diff
@@ -1966,4 +1966,20 @@ export default {
         },
         copyReferralLink: 'Copy referral link',
     },
+    violations: {
+        billableExpense: 'DUMMY -- VIOLATIONS.BILLABLEEXPENSE',
+        categoryOutOfPolicy: 'DUMMY -- VIOLATIONS.CATEGORYOUTOFPOLICY',
+        maxAge: 'DUMMY -- VIOLATIONS.MAXAGE',
+        missingCategory: 'DUMMY -- VIOLATIONS.MISSINGCATEGORY',
+        missingComment: 'DUMMY -- VIOLATIONS.MISSINGCOMMENT',
+        missingTag: 'DUMMY -- VIOLATIONS.MISSINGTAG',
+        overCategoryLimit: 'DUMMY -- VIOLATIONS.OVERCATEGORYLIMIT',
+        overLimit: 'DUMMY -- VIOLATIONS.OVERLIMIT',
+        overLimitAttendee: 'DUMMY -- VIOLATIONS.OVERLIMITATTENDEE',
+        perDayLimit: 'DUMMY -- VIOLATIONS.PERDAYLIMIT',
+        receiptRequired: 'DUMMY -- VIOLATIONS.RECEIPTREQUIRED',
+        tagOutOfPolicy: 'DUMMY -- VIOLATIONS.TAGOUTOFPOLICY',
+        taxOutOfPolicy: 'DUMMY -- VIOLATIONS.TAXOUTOFPOLICY',
+        taxRequired: 'DUMMY -- VIOLATIONS.TAXREQUIRED',
```

Comment:
I think before this got merged to Expensify you'd need to get the final intended copy in here, that's usually part of a PR review?

Reply:
So should I just leave this in and they'll flag it and replace it?

Reply:
@cdanwards knows the process better than me, but I think you should generally start by asking Carlos for the copy